StreamChatUITestsAppUITests/Tests/Base TestCase/StreamTestCase.swift

```diff
@@ -14,24 +14,25 @@ class StreamTestCase: XCTestCase {
     var backendRobot: BackendRobot!
     var participantRobot: ParticipantRobot!
     var server: StreamMockServer!
+    let recordVideo = false
```

Comment:
How can we set the `recoredVideo` to true? (since it's a constant)

Reply:
Nice catch! Updated 👍